dockerfile/dockerfile

```diff
@@ -24,16 +24,16 @@ ARG KUBE_VERSION
 ARG HELM_VERSION
 
 ENV KUBE_VERSION ${KUBE_VERSION:-v1.14.8}
-ENV HELM_VERSION ${HELM_VERSION:-v2.15.0}
-ENV HELM_DIFF_VERSION ${HELM_DIFF_VERSION:-v2.11.0+5}
+ENV HELM_VERSION ${HELM_VERSION:-v3.0.0-rc.1}
```

Comment:
Helm 3 is out, maybe the `rc` part can be removed from `HELM_VERSION`

Reply:
Done 👍